specs/www.rst

```diff
@@ -19,6 +19,8 @@ This spec has had two versions:
 
 * ``2.0``: The first version of the spec, released with ASGI 2.0
 * ``2.1``: Added the ``headers`` key to the WebSocket Accept response
+* ``2.2``: Allow ``None`` in the second item of ``client`` &
+  ``server`` scope values.
```

Comment:
I think this requires a `s/two/three` on line 18. 🤔

Reply:
Thanks